lib/core/src/lib/info-drawer/info-drawer-layout.component.scss

```diff
@@ -14,49 +14,51 @@ $adf-info-drawer-icon-size-half: 24px !default;
 
         overflow: auto;
         width: 100%;
-        background-color: var(--theme-background-color);
+        background-color: var(--theme-card-background-color);
         box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.27);
 
         & .mat-tab-label {
```

Comment:
can we avoid styling mat classes?

Reply:
closing based on https://github.com/Alfresco/alfresco-ng2-components/pull/8995#discussion_r1379736785  comment